sources/packages/web/src/components/students/StudentAppealSharedForm.vue

```diff
@@ -0,0 +1,222 @@
+          multiple
+          :items="applicationAppeals"
+          item-title="description"
+          item-value="formName"
+          v-model="selectedApplicationAppeals"
+          variant="outlined"
+          class="mb-4"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Appeal(s)')]"
+        />
+        <v-select
+          v-if="
+            selectedAppealType === AppealTypes.Other && isOtherOptionAvailable
+          "
+          hide-details="auto"
+          label="Appeal form"
+          density="compact"
+          :items="otherAppeals"
+          item-title="description"
+          item-value="formName"
+          v-model="selectedOtherAppeal"
+          variant="outlined"
+          class="mb-4"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Appeal')]"
+        />
+        <banner
+          v-if="
+            selectedAppealType === AppealTypes.Other && !isOtherOptionAvailable
+          "
+          class="mb-2"
+          type="info"
+          header="Other appeal types will be available for submission soon."
+        />
+      </v-form>
+    </content-group>
+    <footer-buttons
+      class="mt-4"
+      primary-label="Next"
+      justify="end"
+      @primary-click="goToAppealFormsRequests"
+      :show-secondary-button="false"
+      :show-primary-button="selectedAppealType === AppealTypes.Application"
+    />
+  </body-header-container>
+</template>
+<script lang="ts">
+import { useRules, useSnackBar } from "@/composables";
+import { defineComponent, onMounted, ref, watch } from "vue";
+import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
+import { StudentAppealService } from "../../services/StudentAppealService";
+import { EligibleApplicationForAppealAPIOutDTO } from "@/services/http/dto";
+import { useRouter } from "vue-router";
+import { VForm } from "@/types";
+
+enum AppealTypes {
+  Application = "Application",
+  Other = "Other",
+}
+
+interface AppealForm {
+  formName: string;
+  description: string;
+}
+
+export default defineComponent({
+  props: {
+    applicationId: {
+      type: Number,
+      required: false,
+      default: undefined,
+    },
+  },
+  setup(props) {
+    // TODO: Variable to be removed once the "Others" option is implemented.
+    const isOtherOptionAvailable = ref(false);
+    const snackBar = useSnackBar();
+    const router = useRouter();
+    const { checkNullOrEmptyRule } = useRules();
+    const appealsSelectionForm = ref({} as VForm);
+    const eligibleApplications = ref<EligibleApplicationForAppealAPIOutDTO[]>();
+    const loadingEligibleApplications = ref(false);
+    const selectedAppealType = ref<AppealTypes | null>(AppealTypes.Application);
+    const selectedApplicationId = ref<number | null>();
+    const selectedApplicationAppeals = ref<string[]>();
+    const applicationAppeals = ref<AppealForm[]>([
+      {
+        formName: "roomandboardcostsappeal",
+        description: "Room and board costs",
+      },
+    ]);
```

Comment:
The formio form `studentappeals` can be deleted now.

Reply:
I can do it in the upcoming PR alongside E2E tests.